docs/errors/E0711.md

```diff
@@ -0,0 +1,22 @@
+# E0711: missing expression in placeholder within template literal
+
+Everytime you use `${}` within a template literal, which is a string defined
+with backticks, such as this one below, you should specify an expression
+inside its curly braces.
+
+```javascript
+const myString = `Here is a string defined with backticks`;
+```
+
+See an example:
+
+```javascript
+console.log(`My name is ${}`); // error: missing expression in placeholder within template literal
+```
+
+Here is how you can solve it:
+
+```javascript
+const yourName = "Ryan";
+console.log(`My name is ${yourName}`); // Correct
+```
```

Comment:
It is possible that the programmer intended to write `\${}` instead of `${}`. I think it's useful to suggest escaping the `$` or using `""`/`''` instead of backticks.